Gems/ROS2/Code/ros2_target_depends.cmake

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+function(target_depends_on_ros2_packages TARGET_NAME)
+    find_package(ROS2 MODULE)
+    if (NOT ROS2_FOUND)
+        return()
+    endif()
+    #message("Building ROS2 Gem with ros2 $ENV{ROS_DISTRO}")
```

Comment:
You either remove this line if not needed, or enable it and print in  'DEBUG' mode
```
message(DEBUG "Building ROS2 Gem with ros2 $ENV{ROS_DISTRO}")
```